frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/status-card/utils.ts

```diff
@@ -31,14 +31,14 @@ export const getCephHealthState: PrometheusHealthHandler = (responses = [], erro
 };
 
 export const getDataResiliencyState: PrometheusHealthHandler = (responses = [], errors = []) => {
-  if (errors[0]) {
+  const progress: number = getResiliencyProgress(responses[0]);
+  if (errors[0] || Number.isNaN(progress)) {
     return { state: HealthState.UNKNOWN };
   }
   if (!responses[0]) {
     return { state: HealthState.LOADING };
   }
-  const progress = getResiliencyProgress(responses[0]);
-  if (progress && progress < 100) {
+  if (progress < 1) {
```

Comment:
Shouldn't this be `progress >1`?
```suggestion
  if (progress > 1) {
```

Reply:
No, progress needs to be shown for less than 1. the progress is coming as a fraction
e.g. 0.5 < 1
In case of 1<1
Resiliency is ok
Greater than 1 is not a practical case.

Reply:
Ahh ok. I thought it's coming in percentage number.